openmp/libomptarget/include/Shared/Requirements.h

```diff
@@ -33,7 +33,12 @@ enum OpenMPOffloadingRequiresDirFlags : int64_t {
   /// unified_shared_memory clause.
   OMP_REQ_UNIFIED_SHARED_MEMORY = 0x008,
   /// dynamic_allocators clause.
-  OMP_REQ_DYNAMIC_ALLOCATORS = 0x010
+  OMP_REQ_DYNAMIC_ALLOCATORS = 0x010,
+  /// Auto zero-copy extension:
+  /// when running on an APU, the GPU plugin may decide to
+  /// run in zero-copy even though the user did not program
+  /// their application with unified_shared_memory requirement.
+  OMPX_REQ_AUTO_ZERO_COPY = 0x020
```

Comment:
Is this the right place for this? These enumerations are supposed to correspond directly to `omp requires xxx` directives.

Reply:
If you look at the previous review #75999 there was consensus of having this here. I understand this does not match 1-to-1 what OpenMP requires clauses have, but it also means that we do not have to store it in some boolean nor to keep asking the plugin about it.